### ISSUE TITLE
Crash when using epgsql:connect/1 with proplist as argument

### ISSUE THREAD

**Issue by AndreyKlimchuk** [epgsql/epgsql#175]
Bug introduced in latest master (4.2.0)

```
{badmatch,
                          {error,
                           {{function_clause,
                             [{epgsql_cmd_connect,init,
                               [[{size,10},
                                 {host,"..."},
                                 {database,"..."},
                                 {username,"..."},
                                 {password,[]},
                                 {port,5432}]],
                               [{file,"src/commands/epgsql_cmd_connect.erl"},
                                {line,45}]},
                              {epgsql_sock,command_new,4,
                               [{file,"src/epgsql_sock.erl"},{line,256}]},
                              {gen_server,try_handle_call,4,
                               [{file,"gen_server.erl"},{line,661}]},
                              {gen_server,handle_msg,6,
                               [{file,"gen_server.erl"},{line,690}]},
                              {proc_lib,init_p_do_apply,3,
                               [{file,"proc_lib.erl"},{line,249}]}]}
```
Noticed when using pgapp

**Comment by seriyps** [comment]
Hi. Good catch! Will fix

**Comment by seriyps** [comment]
Fixed. Published as 4.2.1

**Comment by AndreyKlimchuk** [comment]
Cool, thanks !
